streams/last_positive/pubspec.lock

```diff
@@ -1,17 +1,348 @@
 # Generated by pub
-# See http://pub.dartlang.org/doc/glossary.html#lockfile
```

Comment:
Weird? Seems to have been run on an old SDK

Reply:
Search for `# See https://www.dartlang.org`

Reply:
I'm in the process of deleting and re-generating all of the `pubspec.lock` files but this line isn't changing.

```
$ pub --version
Pub 2.3.2
```

Reply:
Maybe it hasn't been synced yet. No worries...